Source/WebCore/platform/graphics/avfoundation/objc/MediaPlayerPrivateMediaSourceAVFObjC.mm

```diff
@@ -1431,6 +1431,10 @@ void getSupportedTypes(HashSet<String, ASCIICaseInsensitiveHash>& types) const f
 {
     ASSERT(!m_videoFrameMetadataGatheringObserver || m_synchronizer);
     m_isGatheringVideoFrameMetadata = true;
+    
+#if PLATFORM(COCOA) && !HAVE(AVSAMPLEBUFFERDISPLAYLAYER_COPYDISPLAYEDPIXELBUFFER)
+    willBeAskedToPaintGL();
```

Comment:
`willBeAskedToPaintGL` makes this work as a side effect. I would prefer if we had a new method with a name that made sense for what it does, even if it does exactly the same thing.